### ISSUE TITLE
No Data for all Country's in Self hosted instance. 

### ISSUE THREAD

**Issue by mrghxst** [thedevs-network/kutt#543]
I was wondering if I have to set the MaxMind API Key or somethig to enable Geo tracking because I dont get any locations in my selfhosted instance. And every other tracking works. 
Also in MaxMind Demo all my IP's give Country accurate data.

**Comment by poeti8** [comment]
No, no key is needed, all the data is offline (https://www.npmjs.com/package/geoip-lite). The problem could be that IP address is not properly passed to the Node.js app. For example if you're using NGINX, I suggest you to look how to properly proxy IP address. You can check out [Kutt.it's NGINX config](https://github.com/thedevs-network/kutt/wiki/NGINX-config) to see how it's done. It's the `proxy_set_header  X-Real-IP  $remote_addr;` line.
